src/benchmarks/gc/docs/bench_file.md

```diff
@@ -36,6 +36,9 @@ benchmarks:
       tlgb: 1
       lohar: 0
       pohar: 0
+      sohsr: 10-4000
```

Comment:
do you want to also change the name of this profile from nosurvive to something like surv_2percent? 'cause it's clearly surviving... seems odd to call it nosurvive.